tests/cover/test_cathetus.py

```diff
@@ -20,63 +20,27 @@
 import sys
 import math
 
-from hypothesis.internal.cathetus import cathetus
-
-
-def assert_cathetus_exact(h, a, b):
-    b0 = cathetus(h, a)
-    assert b == b0, (
-        'expected cathetus(%g, %g) == %g, got %g' %
-        (h, a, b, b0)
-    )
-
-
-def assert_cathetus_nan(h, a):
-    b = cathetus(h, a)
-    assert math.isnan(b), (
-        'expected cathetus(%g, %g) == NaN, got %g' %
-        (h, a, b)
-    )
+import pytest
 
-
-def assert_cathetus_inf(h, a):
-    b = cathetus(h, a)
-    assert math.isinf(b), (
-        'expected cathetus(%g, %g) == Infinity, got %g' %
-        (h, a, b)
-    )
+from hypothesis.internal.cathetus import cathetus
 
 
 def test_cathetus_subnormal_underflow():
     u = sys.float_info.min * sys.float_info.epsilon
     h = 5 * u
     a = 4 * u
     b = cathetus(h, a)
-    assert not math.isnan(b), (
-        'expecting cathetus(%g, %g) not NaN, got %g' %
-        (h, a, b)
-    )
-    assert b > 0, (
-        'expecting cathetus(%g, %g) positive, got %g' %
-        (h, a, b)
-    )
     assert b == 3 * u, (
-        'expecting cathetus(%g, %g) == %g, got %g' %
-        (h, a, 3 * u, b)
+        'expecting cathetus(%g, %g) == %g, got %g' % (h, a, 3 * u, b)
```

Comment:
I think this could be reduced further to:

```python
assert cathetus(h, a) == 3 * u
```

no?

Reply:
I've reduced this one, but for the others if it ever fails I'm going to want the explicit note about what was expected (which is distinct from reporting what we got, as pytest can do automatically).

Reply:
Yep, seems reasonable.